test/utils.h

```diff
@@ -70,6 +70,14 @@ inline TensorView* makeSymbolicTensor(
   return TensorViewBuilder().shape(shape).dtype(dtype).build();
 }
 
+// allows overload resolution with size-1 initializer list
+inline TensorView* makeSymbolicTensor(
+    std::initializer_list<int64_t> shape,
```

Comment:
This API is just added to allow overload of `makeSymbolicTensor({-1}, ...)`, which would otherwise be called into `makeSymbolicTensor(size_t, ...)`

Reply:
note to myself. Do a quick clean up for other APIs as well!

Reply:
C++ syntax is weird...